app/src/main/java/cm/aptoide/pt/download/DownloadInstallationProvider.java

```diff
@@ -28,6 +29,7 @@
  */
 public class DownloadInstallationProvider implements InstallationProvider {
 
+  private static final String TAG = "AptoideDownloadManager";
```

Comment:
The tag doesn't match the class name. It refers to another class. Mistake maybe?